tests/utils.go

```diff
@@ -203,16 +204,14 @@ func StopVirtualMachineWithTimeout(vm *v1.VirtualMachine, timeout time.Duration)
 	By("Stopping the VirtualMachineInstance")
 	virtClient := kubevirt.Client()
 
-	Eventually(func() error {
-		updatedVM, err := virtClient.VirtualMachine(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
-		Expect(err).ToNot(HaveOccurred())
-		updatedVM.Spec.Running = pointer.P(false)
-		updatedVM.Spec.RunStrategy = nil
-		_, err = virtClient.VirtualMachine(updatedVM.Namespace).Update(context.Background(), updatedVM, metav1.UpdateOptions{})
-		return err
-	}, timeout, 1*time.Second).ShouldNot(HaveOccurred())
-	updatedVM, err := virtClient.VirtualMachine(vm.Namespace).Get(context.Background(), vm.Name, metav1.GetOptions{})
+	p, err := patch.New(
+		patch.WithAdd("/spec/running", nil),
+		patch.WithAdd("/spec/runStrategy", v1.RunStrategyHalted),
```

Comment:
IIUC this change includes the swap between using `running` and `runStrategy`, but nothing has been mentioned in the commit message.

Could you please explicitly mention it there?
It would be even better if a separate commit would have changed it, but I will not insist.

Reply:
It's outdated and dropped by the following commit, so I'll squash them.